extensions/amp-adblock-detector/1.0/test/validator-amp-adblock-detector.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Please add a copyright statement at the top of this file as well as a description of the test in a comment. See other components for examples.

Reply:
Scratch that, this requirement has changed.